src/components/Navigation/NavigationList/NavigationList.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { px } from '../../../utils';
+import classNames from 'classnames';
+import { NavigationItemProps } from '../NavigationItem/NavigationItem';
+import { HeaderContext } from '../../Header/Header';
+
+export type NavigationListProps = React.HTMLAttributes<HTMLElement>;
+
+const NavigationList = ({ children, className }: React.PropsWithChildren<NavigationListProps>) => {
+  const { expanded } = React.useContext(HeaderContext);
+  const largeCtaItems: React.ReactNode[] = [];
+  const smallCtaItems: React.ReactNode[] = [];
+  React.Children.map(
+    children as React.ReactElement<NavigationItemProps>[],
+    (child: React.ReactElement<NavigationItemProps>) => {
+      if (child && child.props.navGroup === 'nav-link-lg') {
+        largeCtaItems.push(child);
```

Comment:
What do you think about rather than do a `.push` here, doing a `const largeCtaItems = React.Children.toArray(children).filter(....)`?  Also you need to check that the child is a valid React element in the filter (should be able to remove the direct type casting with `as ReactElement` if you do that...)